spec/csv_row_model/validators/date_time_format_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'spec_helper'
+
+describe DateTimeFormatValidator do
+  let(:klass) do
+    Class.new do
+      include ActiveWarnings
+      attr_accessor :string1
+      warnings do
+        validates :string1, date_format: true
```

Comment:
it's a magical thing that `ActiveModel::Validations` does. it hooks on to the class name. sorry, i mean `date_time_format: true`